specs/binding-http.spec/src/main/scripts/io/aklivity/zilla/specs/binding/http/streams/network/rfc7540/connection.management/server.sent.end.before.response.received/client.rpt

```diff
@@ -0,0 +1,76 @@
+#
```

Comment:
Rename to `server.sent.close.before.response.headers` ?

Reply:
Done